cmd/collector/app/metrics.go

```diff
@@ -237,6 +241,9 @@ func (m *countsBySvc) countByServiceName(serviceName string, isDebug bool) {
 			debugStr = "true"
 		}
 		tags := map[string]string{"svc": serviceName, "debug": debugStr}
+		if samplerType != "" {
```

Comment:
Only add tag when `samplerType` is not empty. This ensures only trace counter gets this tag.

Reply:
can you add this comment in the code? This feels slightly hacky and I think we can refactor this better (not sure how atm)